modules/tinymce/src/core/main/ts/content/InsertContentImpl.ts

```diff
@@ -24,16 +24,17 @@ import * as InsertList from './InsertList';
 
 const wrappedElements = [ 'pre' ];
 
-const shouldPasteContentOnly = (fragment: AstNode, parentNode: Element) => {
+const shouldPasteContentOnly = (dom: DOMUtils, fragment: AstNode, parentNode: Element) => {
   const firstNode = fragment.firstChild as AstNode;
   const lastNode = fragment.lastChild as AstNode;
+  const last = lastNode.attr('data-mce-type') === 'bookmark' ? lastNode.prev : lastNode;
 
+  const isCopyingOnlyOneTag = firstNode === last;
   const isWrappedElement = Arr.contains(wrappedElements, firstNode.name);
   const isPastingInTheSameTag = firstNode.name === parentNode.tagName.toLowerCase();
-  const last = lastNode.attr('data-mce-type') === 'bookmark' ? lastNode.prev : lastNode;
-  const isCopingOnlyOneTag = firstNode === last;
+  const isContentEditableTrue = dom.getContentEditable(parentNode) !== 'false';
```

Comment:
Would like some clarification or suggestion for a better way to ensure this variable is false when pasting on cef element (`parentNode`), or perhaps in the `insertHtmlAtCaret` method.